centreon/build.conf

```diff
@@ -22,7 +22,8 @@ plugin-web-description=<p>Plugin for Centreon integartion with Rudder</p>
 # - x.y(.z): plugin major.minor.micro. Micro should be omitted. When omitted, z is assumed to be 0. 
 # For the build, we split the information between two properties, rudder branch and plugin version, 
 # which must be concaneted with "-" to build the plugin version. 
-plugin-branch=1.5-SNAPSHOT
+plugin-branch=1.5
+plugin-branch-nightly=I.5
```

Comment:
I -> 1

Reply:
This, or we decide to switch to roman numerals for nightlies

Reply:
RUDDER VI.II.III